docs/class-definitions.md

```diff
@@ -154,66 +162,122 @@ await space.updateProfileData((oldProfileData) => {
 })
 ```
 
+# Members
+
+Handles members within a space.
+
+## Methods
+
 ### subscribe
 
-Listen to events for the space. See [EventEmitter](/docs/usage.md#event-emitters) for overloading usage.
+Listen to member events for the space. See [EventEmitter](/docs/usage.md#event-emitters) for overloading usage.
 
 Available events:
 
-- #### **membersUpdate**
+- #### **update**
 
-  Listen to updates to members.
+  Listen to all updates for members in a space.
 
   ```ts
-  space.subscribe('membersUpdate', (members: SpaceMember[]) => {});
+  space.members.subscribe('update', (member: SpaceMember) => {});
+  ```
+
+  This is the same as not specifying the event:
+
+  ```ts
+  space.members.subscribe((member: SpaceMember) => {})
+
   ```
 
   Triggers on:
-  - presence updates ([`enter`, `leave`, `update` and `present` events](https://ably.com/docs/presence-occupancy/presence?lang=javascript))
+  - presence updates ([`enter`, `leave`, `remove`, `update` events](https://ably.com/docs/presence-occupancy/presence?lang=javascript))
```

Comment:
There is no presence `remove` event, the `remove` event is a Spaces SDK specific event emitted 120s after a `leave` event without a subsequent `enter` or `update` for the same member, so I think it's worth its own bullet point.

Reply:
Saying that, I see this is in the section describing the behaviour of the members `update` event, which is emitted [when a member's profile data changes](https://github.com/ably-labs/spaces/blob/main/src/Members.ts#L55-L60),  so the description I give above belongs in the section describing the `remove` event, not the `update` event, and this section should say the `update` event is triggered on profile data changes.

Reply:
> I understood it to be subscribing to all updates

So `space.subscribe('update', ...)` subscribes to all updates within a space (i.e. whether its a profile data change, a location change, or a lock change, basically any `enter`, `update`, or `leave` presence event).

`space.members.subscribe('update', ...)` is however _just_ for profile data changes, with location changes emitted by `space.locations.subscribe` and lock changes by `space.locks.subscribe`.

Given your confusion though, perhaps this is not a particularly intuitive way to divide the events across the different classes (although I think something to discuss separately, let's just focus on describing the current behaviour here).